css-floatgrid.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Floatgrid</title>
+    <style type="text/css">
+    @import "https://yarnpkg.com/en/package/normalize.css";
+    .grid {
+      background-color: #eee;
+      border:1px solid #000;
+      margin: auto;
+    }
+    .col {
+      float:left;
+      background-color: #eee;
+/*      padding-right: 10px;*/
```

Comment:
Remove unused, commented out code that's not a comment.

Reply:
complete